lbz/resource.py

```diff
@@ -207,6 +207,8 @@ def get_pagination(self, total_items: int, limit: int, offset: int) -> dict:
 
     @property
     def _pagination_uri(self) -> str:
-        if query_params := copy_without_keys(self.request.query_params, "offset", "limit"):
-            return f"{self.urn}?{urlencode(query_params)}&offset={{offset}}&limit={{limit}}"
+        if query_params := self.request.query_params.clean_keys(["offset", "limit"]):
+            # TODO: Make MultiDict be a Mapping not a object to remove type ignore below
+            encoded_params = urlencode(query_params)  # type: ignore
```

Comment:
When you improve the code around `MultiDict.items()`, you should start using the `doseq=True` argument to put each value separately to the URL 🙂